pom.xml

```diff
@@ -60,11 +60,16 @@
         <version.camel.qpid.proton>0.26.0</version.camel.qpid.proton>
 
         <!-- WildFly versions -->
-        <version.wildfly>12.0.0.Final</version.wildfly>
+        <version.wildfly>13.0.0.Final</version.wildfly>
         <version.wildfly.arquillian>2.1.0.Final</version.wildfly.arquillian>
-        <version.wildfly.cxf>3.2.2</version.wildfly.cxf>
-        <version.wildfly.fasterxml.jackson>2.9.4</version.wildfly.fasterxml.jackson>
-        <version.wildfly.infinispan>9.1.6.Final</version.wildfly.infinispan>
+        <version.wildfly.cxf>3.2.4-jbossorg-1</version.wildfly.cxf>
+        <version.wildfly.fasterxml.jackson.core>2.9.5</version.wildfly.fasterxml.jackson.core>
+        <version.wildfly.fasterxml.jackson.datatype>2.9.5</version.wildfly.fasterxml.jackson.datatype>
+        <version.wildfly.fasterxml.jackson.jaxrs>2.9.5</version.wildfly.fasterxml.jackson.jaxrs>
+        <version.wildfly.fasterxml.jackson.module>2.9.5</version.wildfly.fasterxml.jackson.module>
+        <version.com.fasterxml.jackson.dataformat>${version.wildfly.fasterxml.jackson.core}</version.com.fasterxml.jackson.dataformat>
```

Comment:
This version redirect should not be necessary. We like to stick with 'version.wildfly.fasterxml.jackson.core'. In the unlikely case that wildfly diverges versions for datatype, jaxrs, module or dataformat from core, we can introduce additional properties (beside the one for core), which must then map to some property in some wf pom (i.e. the one that defines the divergence.